### PR TITLE
[zh] Sync 1.24 concepts-4

### PR DESCRIPTION
**Pull request by mengjiao-liu**
related issue: #33476 #33955
concepts 4
```
75	24	content/en/docs/concepts/services-networking/service.md
36	16	content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md
15	0	content/en/docs/concepts/workloads/controllers/cron-jobs.md
```

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 1]
```diff
@@ -123,29 +118,80 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
...
+  - When the number of eligible domains with matching topology keys equals or is greater than 
+    `minDomains`, this value has no effect on scheduling.
+  - When `minDomains` is nil, the constraint behaves as if `minDomains` is 1.
+  - When `minDomains` is not nil, the value of `whenUnsatisfiable` must be "`DoNotSchedule`".
+-->
+- **minDomains** 表示符合条件的域的最小数量。域是拓扑的一个特定实例。
+  符合条件的域是其节点与节点选择器匹配的域。
+
+  - 指定的 `minDomains` 的值必须大于 0。
+  - 当符合条件的具有匹配拓扑键的域的数量小于 `minDomains` 时，Pod 拓扑传播将“全局最小值”（global minimum） 设为
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:156]:
```suggestion
  - 当符合条件的、拓扑键匹配的域的数量小于 `minDomains` 时，Pod 拓扑分布将“全局最小值”（global minimum） 设为
```

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 2]
```diff
@@ -123,29 +118,80 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
...
+
+  {{< note >}}
+  <!--
+  The `minDomains` field is an alpha field added in 1.24. You have to enable the
+  `MinDomainsInPodToplogySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  in order to use it.
+  -->
+  The `minDomains` field is an alpha field added in 1.24. You have to enable the
+  `MinDomainsInPodToplogySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  in order to use it.
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:172]: Remove the above lines

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 3]
```diff
@@ -511,35 +557,27 @@ profiles:
...
-using default constraints for `PodTopologySpread`.
+[`SelectorSpread` plugin](/docs/reference/scheduling/config/#scheduling-plugins)
+is disabled by default. It's recommended to use `PodTopologySpread` to achieve similar
+behavior.
 -->
-默认调度约束所生成的评分可能与
-[`SelectorSpread` 插件](/zh/docs/reference/scheduling/config/#scheduling-plugins)
-所生成的评分有冲突。
-建议你在为 `PodTopologySpread` 设置默认约束是禁用调度方案中的该插件。
+[`SelectorSpread` 插件](/zh/docs/reference/scheduling/config/#scheduling-plugins) 默认是禁用的。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:564]:
```suggestion
[`SelectorSpread` 插件](/zh/docs/reference/scheduling/config/#scheduling-plugins)默认是被禁用的。
```

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 4]
```diff
@@ -511,35 +557,27 @@ profiles:
...
-kube-scheduler uses the following default topology constraints for the
-`PodTopologySpread` plugin configuration:
+If you don't configure any cluster-level default constraints for pod topology spreading,
+then kube-scheduler acts as if you specified the following default topology constraints:
 -->
-当你使用了默认启用的 `DefaultPodTopologySpread` 特性门控时，原来的
-`SelectorSpread` 插件会被禁用。
-kube-scheduler 会使用下面的默认拓扑约束作为 `PodTopologySpread` 插件的
-配置：
+如果你没有为 Pod 拓扑扩展配置任何集群级别的默认约束，那么
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:579]:
```suggestion
如果你没有为 Pod 拓扑分布配置任何集群级别的默认约束，
```

**Comment by mengjiao-liu** [comment]
All comments addressed.

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 5]
```diff
@@ -123,29 +118,77 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
...
+  - 当 `minDomains` 不为 nil 时，`whenUnsatisfiable` 的值必须为 "`DoNotSchedule`" 。
+
+  {{< note >}}
+  <!--
+  The `minDomains` field is an alpha field added in 1.24. You have to enable the
+  `MinDomainsInPodToplogySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  in order to use it.
+  -->
+  `minDomains` 字段是在 1.24 版本中新增的 alpha 字段。你必须启用
+  `MinDomainsInPodToplogySpread`[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)才能使用它。
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:171]: nit: Missing a space.
```suggestion
  `MinDomainsInPodToplogySpread` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)才能使用它。
```

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 6]
```diff
@@ -123,29 +118,77 @@ You can define one or multiple `topologySpreadConstraint` to instruct the kube-s
...
+  符合条件的域是其节点与节点选择器匹配的域。
+
+  - 指定的 `minDomains` 的值必须大于 0。
+  - 当符合条件的、拓扑键匹配的域的数量小于 `minDomains` 时，Pod 拓扑分布将“全局最小值”（global minimum） 设为
+    0，然后进行 `skew` 计算。“全局最小值”是一个符合条件的域中匹配 Pods 的最小数量，
+    如果符合条件的域的数量小于 `minDomains`，则全局最小值为零。
+  - 当符合条件的拓扑键匹配域的个数等于或大于 `minDomains` 时，该值对调度没有影响。
+  - When `minDomains` is nil, the constraint behaves as if `minDomains` is 1.
+  - 当 `minDomains` 为 nil 时，约束的行为等于 `minDomains` 为 1。
+  - 当 `minDomains` 不为 nil 时，`whenUnsatisfiable` 的值必须为 "`DoNotSchedule`" 。
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:162]:
```suggestion
  - 当符合条件的拓扑键匹配域的个数等于或大于 `minDomains` 时，该值对调度没有影响。
  - 当 `minDomains` 为 nil 时，约束的行为等于 `minDomains` 为 1。
  - 当 `minDomains` 不为 nil 时，`whenUnsatisfiable` 的值必须为 "`DoNotSchedule`" 。
```

Comment (mengjiao-liu) [reply content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md]: Updated. Thanks.

**Pull request merged**